core/crypto/crypto.cpp

```diff
@@ -228,7 +228,7 @@ String ResourceFormatLoaderCrypto::get_resource_type(const String &p_path) const
 	return "";
 }
 
-Error ResourceFormatSaverCrypto::save(const Ref<Resource> &p_resource, const String &p_path, uint32_t p_flags) {
+Error ResourceFormatSaverCrypto::save(const Ref<Resource> &p_resource, const String &p_path, uint32_t p_flags, const int p_integrity_level) {
```

Comment:
```suggestion
Error ResourceFormatSaverCrypto::save(const Ref<Resource> &p_resource, const String &p_path, uint32_t p_flags, int p_integrity_level) {
```

Usually we do not use `const` for primitive types, also any reason it's not using `FileAccess::SaveIntegrityLevel p_integrity_level)` directly?

Reply:
Ah! I missed this one. Removed. Thanks.